docs/setup.md

```diff
@@ -16,7 +16,7 @@ pnpm install --save-dev eslint
 # install airbnb eslint config + peer dependencies
 pnpm install --save-dev eslint-config-airbnb eslint-config-airbnb-typescript eslint-plugin-import eslint-plugin-jsx-a11y eslint-plugin-react eslint-plugin-react-hooks
```

Comment:
@jimbrig reasoning for using `airbnb` versions?

Reply:
I found it to be the best setup and most inline with what our standards are.

Reference:

- https://www.npmjs.com/package/eslint-config-airbnb
- https://github.com/airbnb/javascript 
- https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb